src/main/java/net/mcreator/blockly/java/blocks/TextJoinBlock.java

```diff
@@ -41,29 +42,39 @@
 				return;
 			}
 
+			List<String> inputCodes = new ArrayList<>();
+			for (int i = 0; i < sumnum; i++) {
+				inputCodes.add("null");
+				for (Element element : elements) {
+					if (element.getAttribute("name").equals("ADD" + i)) {
+						inputCodes.set(i, BlocklyToCode.directProcessOutputBlock(master, element));
+						break;
+					}
+				}
+				if (inputCodes.get(i).equals("null"))

```

Comment:
You should not add null inside, but instead check if list here is empty. A cleaner solution I think.